packages/backend/src/plugins/catalog.ts

```diff
@@ -26,6 +27,19 @@ export default async function createPlugin(
     }),
   );
 
+  builder.addEntityProvider(
+    GithubEntityProvider.fromConfig(env.config, {
+      logger: env.logger,
+      // optional: alternatively, use scheduler with schedule defined in app-config.yaml
+      schedule: env.scheduler.createScheduledTaskRunner({
+        frequency: { minutes: 30 },
+        timeout: { minutes: 3 },
```

Comment:
```suggestion
        timeout: { minutes: 3 },
        initialDelay: { minutes: 1 },
```
Time value is flexible 

The amount of time that should pass before the first invocation happens. This can be useful in cold start scenarios to stagger or delay some heavy compute jobs. If no value is given for this field then the first invocation will happen as soon as possible according to the cadence.